debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/ExportStatus.java

```diff
@@ -300,6 +304,30 @@ public void updateQueueSegmentMetaInfo(long segmentNo, long committedSize, Map<P
 
     }
 
+    public boolean checkIfSwitchOperationRequested(String operation) throws SQLException {
+        Statement selectStmt = metadataDBConn.createStatement();
+        String query = String.format("SELECT json_text from %s where key = '%s'",
```

Comment:
Probably worth creating a separate func for getting the MigrationStatusRecordJson from metadb. Could be useful for future use-cases.

Reply:
done